Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -13,6 +13,23 @@ final class SubscriptionsViewController: BaseViewController {
 
     @IBOutlet weak var tableView: UITableView!
     @IBOutlet weak var activityViewSearching: UIActivityIndicatorView!
+    @IBOutlet weak var avatarViewContainer: AvatarView! {
+        didSet {
+            if let avatarView = AvatarView.instantiateFromNib() {
+                self.avatarView.frame = avatarViewContainer.bounds
```

Comment:
This isn't actually changing the `self.avatarView` right? Because it's nil at this point.